packages/sitecore-pipelines/package.json

```diff
@@ -12,7 +12,7 @@
     "test": "mocha --require ts-node/register \"./src/**/*.test.ts\"",
     "prepublishOnly": "npm run build",
     "runner": "npm run build && node ./dist/runner",
-    "coverage": "nyc --require ts-node/register npm test"
+    "coverage": "nyc npm test"
```

Comment:
When running this locally, Lerna says it's attempting to run `test` in 18 packages but there are only 17, and the process does not exit after the 17th package's tests are complete.